src/ios/APPLocalNotification.m

```diff
@@ -735,4 +756,31 @@ - (void) fireEvent:(NSString*)event notification:(UILocalNotification*)notificat
     }
 }
 
+ /**
+ * Get notification identifier to send to JS.
+ */
+ - (void) handleNotificationAction:(NSNotification*)notification
+ {
+     NSString* identifier = [notification object];
+     
+     NSDictionary* userInfo = notification.userInfo;
+     UILocalNotification *localNotification = [userInfo objectForKey:@"localNotification"];
+     
+     NSDictionary* responseInfo = [userInfo objectForKey:@"responseInfo"];
+     
+     NSDictionary* dataDict = [NSDictionary dictionaryWithObjectsAndKeys:identifier, @"identifier",
+                               [responseInfo objectForKey:@"UIUserNotificationActionResponseTypedTextKey"], @"responseInfoText", nil];
+     NSData* jsonData = [NSJSONSerialization dataWithJSONObject:dataDict options:0 error:nil];
```

Comment:
Looks like the length is going over 80 in lots of places, should this be fixed in all of my obj-c code?